cheerlights/app.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import . "github.com/alexellis/blinkt_go"
```

Comment:
Are you OK to switch this over to the sysfs library (you could keep it in this directory structure)?

That will mean we can use this as a swarm service (which will be a cool demo!)